lua/illuminate/providers/lsp.lua

```diff
@@ -48,14 +48,19 @@ end
 
 function M.is_ready(bufnr)
     local supported = false
-    if vim.lsp.for_each_buffer_client then
+    if vim.fn.has('nvim-0.10') ~= 1 and vim.lsp.for_each_buffer_client then
         supported = false
         vim.lsp.for_each_buffer_client(bufnr, function(client)
             if client and client.supports_method('textDocument/documentHighlight') then
                 supported = true
             end
         end)
     end
+    for _, client in pairs(vim.lsp.get_clients(bufnr)) do
```

Comment:
Without that change this gives an error.
```suggestion
    for _, client in pairs(vim.lsp.get_clients({bufnr = bufnr})) do
```

Reply:
Thanks